backend/tracim_backend/command/periodic.py

```diff
@@ -103,7 +104,12 @@ def take_app_action(self, parsed_args: argparse.Namespace, app_context: AppEnvir
         user_api = UserApi(current_user=None, session=session, config=config)
 
         for user in user_api.get_all():
-            if not user.email:
+            if (
+                not user.email
+                or not user.is_active
+                or user.is_deleted
+                or user.auth_type == AuthType.UNKNOWN
+            ):
```

Comment:
I suggest refactoring this into a function into User to look like : 

```python
if not user.can_receive_summary_mail():
    continue
```

Multiple benefits : 
 - Readability from this file
 - Only one modification zone for determining what is required to be able to send a sommary mail to an user